engine/src/test/java/org/hibernate/validator/testutils/ConstraintValidatorInitializationHelper.java

```diff
@@ -87,4 +87,32 @@ private ConstraintValidatorInitializationHelper() {
 	public static HibernateConstraintValidatorInitializationContext getDummyConstraintValidatorInitializationContext() {
 		return DUMMY_CONSTRAINT_VALIDATOR_INITIALIZATION_CONTEXT;
 	}
+
+	public static HibernateConstraintValidatorInitializationContext getConstraintValidatorInitializationContext(
```

Comment:
With this method added we could initialize the `DUMMY_CONSTRAINT_VALIDATOR_INITIALIZATION_CONTEXT` above with it. Might look cleaner with just:
```java
private static final HibernateConstraintValidatorInitializationContext DUMMY_CONSTRAINT_VALIDATOR_INITIALIZATION_CONTEXT
			= getConstraintValidatorInitializationContext( new DefaultScriptEvaluatorFactory( null ), DefaultClockProvider.INSTANCE, Duration.ZERO, null );
```

Reply:
Yup, good catch, done.